birdhouse/birdhouse-compose.sh

```diff
@@ -118,7 +118,9 @@ fi
 COMPOSE_EXTRA_OPTS=""
 
 if [ x"$1" = x"up" ] || [ x"$1" = x"restart" ]; then
-  COMPOSE_EXTRA_OPTS="${BIRDHOUSE_COMPOSE_UP_EXTRA_OPTS}"
+  if [ x"$1" = x"up" ]; then
```

Comment:
Can you move this out of the outer if statement. Otherwise it makes it seem like this is directly related to the pre-compose-up scripts.

Reply:
done https://github.com/bird-house/birdhouse-deploy/pull/515/commits/3a2732fb72759a87e8667ed7fac79c9004438189